tuna/miopen/subcmd/import_configs.py

```diff
@@ -154,6 +154,10 @@ def parse_line(args: argparse.Namespace, line: str, counts: dict,
     driver = DriverBatchNorm(line, args.command)
   else:
     driver = DriverConvolution(line, args.command)
+    if not driver.out_layout == driver.in_layout == driver.fil_layout:
+      raise ValueError(
+          f"Non-matching layouts {driver.out_layout}, {driver.in_layout}, {driver.fil_layout}"
+      )
```

Comment:
This might not always be true, since the filter layout for NHWC convolution is CNHW ?

Reply:
Lets confirm by running some driver commands